torch/_dynamo/variables/misc.py

```diff
@@ -1220,6 +1220,12 @@ def get_np_to_tnp_map():
     return np_fn_to_tnp_fn
 
 
+@functools.lru_cache(maxsize=1)
+def get_tnp_to_np_map():
```

Comment:
nit: As someone unfamiliar with this stuff this function is extraordinarily mysterious lol

Reply:
Added a docstring.